src/pilot/assign_role.py

```diff
@@ -381,18 +385,39 @@ def get_raid_controller_physical_disk_ids(drac_client, raid_controller_fqdd):
         key=physical_disk_id_to_key)
 
 
-def define_storage_logical_disks(drac_client, raid_controller_name):
+def define_storage_logical_disks(drac_client, raid_controller_ids):
+    boss_controller = None
+    raid_controller = None
     all_physical_disks = drac_client.list_physical_disks()
 
+    if len(raid_controller_ids) > 1:
+        for cntrl in raid_controller_ids:
+            if drac_client.is_boss_controller(cntrl):
+                boss_controller = cntrl
+            else:
+                raid_controller = cntrl
```

Comment:
Here, we could add cntrl to the raid_controllers list

Reply:
Done